docs/config.ini

```diff
@@ -18,29 +18,29 @@ default = index.html
 home = getting_started/get_started.html
 outline = developing_course/course_outline.html
 unit = developing_course/course_units.html
-updates = building_course/handouts_updates.html
-pages = building_course/pages.html
-files = building_course/course_files.html
-textbooks = building_course/textbooks.html
-schedule = building_course/setting_up_student_view.html
-grading = building_course/establish_grading_policy.html
-team_course = building_course/creating_new_course.html#add-course-team-members
+updates = course_assets/handouts_updates.html
+pages = course_assets/pages.html
+files = course_assets/course_files.html
+textbooks = course_assets/textbooks.html
+schedule = set_up_course/setting_up_student_view.html
+grading = grading/index.html
+team_course = set_up_course/course_staffing.html#add-course-team-members
 team_library = creating_content/libraries.html#give-other-users-access-to-your-library
```

Comment:
@catong a lucky chance -- I already had the open_edx html open in my browser, so I was using that to verify each of the new pages. Kind of on a whim, I decided it made sense to test everything, and not just what changed in this commit, by inputting these URL endings in (!)
Might make sense to have Peter or Sylvia do the same thing again, maybe one testing each of the guides?